mmv1/products/bigquery/Dataset.yaml

```diff
@@ -83,11 +86,6 @@ examples:
       dataset_id: 'example_dataset'
       account_name: 'bqowner'
     exclude_docs: true
-  - name: 'bigquery_dataset_external_reference_aws_test'
```

Comment:
What is the reason to convert this test to the handwritten test?

Reply:
the test was failing due to a diff in dataset `etag`/`last_modified_time` ([error here](https://ffa2a4d2296c2beebd95ce7dbb9d4fbd4766184c287090cb1d6442d-apidata.googleusercontent.com/download/storage/v1/b/ci-vcr-logs/o/beta%2Frefs%2Fheads%2Fauto-pr-12475%2Fartifacts%2Fa4ff2d7f-b98e-4012-8e77-2802ec2683f6%2Fbuild-log%2Frecording_build%2FTestAccBigQueryDataset_bigqueryDatasetExternalReferenceAwsTestExample_recording_test.log?jk=ASCxOPsQKzn7567Gal2tChl7bdIauo1wuEqleQHIteXLqjrfMi_II5K55HLCJHBjZuZdHaIPVEIMWAjI7qC08hYEJ_3kpnSdu67FOrxcMccUiR2-rp75zD1BmS1wMQTjUfSS-tzDpnSl-AF0_YC2vs7UDxJ797fYoxZ2tF7m9p4Gm8rHcFsPpA_VEBfhqX0ZiK8xncJbV_zl3u9FfuKrfeubri_OFZyF0lYlsuftGdMMn2CqTbzRZtkI5fG1ix1vZbfe5_E3o7ynsejeUDXre6mqVVb829L308RCmuVQwEvRFLdokQk_DMVg2PxEE36aiA79Z_MF8CMDdKXcdF1tBCKEN5CBBUbQ5enkR2GXkQljCQQ2n-UpwiwZo3Io6asRCvkJtzFTrEmRp2wNNGNPDhyiYLHaI2OIJLl7m2iNGN_3URGKarCuSvXfO5zhbO87tWc4i1XMw9gchEOMbXTGhdyR_BHBqqgEcu3Pj1MpZpnynJ0NDsmuDO5DbpmsAExYyV8kUQMfdkqEUZfa11kcpMf4laFgkDdtWqMi3O0xXxOxxVPntuAN-ExJ76jSYuo3DSKiCjoLdhWl1rb7OR7PsAuPXo_BBX0DjuH-GLE8kpVnHYbkcx3Pl2UfFirXFKNHbkH4tapjiW7izZKRC7Co3EgYPkGeZOgD2kl4of2UW3eu9IU8091ipDJIBeiKnSvMcXj4a4YWvIcZINZfAhTgMYour5nBOyCOG9pcH3a_ToyTpFUXBDar7c44HjVD8ljreHPqc70B0btIQsJIqUBANAu9rzvjTWFDs1_Lv7o32sEsF0JA1_M3uSN3yX28RJnJidQvdIpmExdXfSGcqh3SHxzNP_9-xF_zltZ3wgvIrz2ZeA1KIfkyQniQDqaPqs157EgCu6m4at2Ik1Wae1kDN_sxCInsrSrVIPhRgaHg1gVSkMfHWhwF3lrjcfLragaKLskQycHlAVxfnB6knR54nb2u0HIuD4sYoHXu4VUMLh8TYsm_WfacNqMxGxVW_q39d0X8XPNPFdZhMF2IKPCSvkhBK4WVSpNKeIqnJZLM9T6wbcWu73Hj0OB-mmaAfFXBxbpVm8RLgbKklxgC1FQTVZ9J2R8gpUqGMWv99VKlTt35ecVt9YK249NxL719U3gSxpd9Yx1UVTPCHJSgUlpoe4nLk_dNrL5gqzFklQiYgg3aOKxeKy7s7eoALmtmmzhFHxr-5RkAyDK7fz8igLKEuXJPKgdx1ndC5lq5eCG6UpoU8OR4FP5vQ099HT3QOmE9conxyn0KK3nIQLmaOFOII7ypXPFw_4Ybq8J5tQ95nC5xu_t547928ZS3S0a4XABJZ5C1Zm1PhepLGU0Ucn9Xk941B6_0MvrFUoYZfayCkrNRrkm1WA&isca=1)). Changed the test to a handwritten one and added the attributes to `ImportStateVerifyIgnore`.